src/Analysis/GWASV2/Shared/Covariates/Covariates.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+// import '../GWASCovariates.css';
+import PropTypes from 'prop-types';
+import { useQuery } from 'react-query';
+import { Table, Spin } from 'antd';
+import { fetchCovariates, queryConfig } from '../wizardEndpoints/cohortMiddlewareApi';
+import { useFetch } from '../formHooks';
+import { useSourceContext } from '../Source';
+
+const Covariates = ({
+  // searchTerm,
+  selected,
+  handleSelect,
+}) => {
+  const { source } = useSourceContext();
+  const covariates = useQuery(['covariates', source], () => fetchCovariates(source), queryConfig);
+  const fetchedCovariates = useFetch(covariates, 'concepts');
+  // const displayedCovariates = useFilter(fetchedCovariates, searchTerm, 'concept_name');
+
+  const covariateSelection = () => ({
+    type: 'radio',
+    columnTitle: 'Select',
+    selectedRowKeys: (selected) ? [selected.concept_id] : [],
+    onChange: (_, selectedRows) => {
+      handleSelect(selectedRows[0]);
+    },
+    getCheckboxProps: (record) => ({
```

Comment:
this can be removed, since we use radio now.

Reply:
I deleted this block of code: 

```
    getCheckboxProps: (record) => ({
      disabled: record.size === 0, // || selected.some((a) => a.concept_id === record.concept_id)
    }),
   ```